### ISSUE TITLE
Can't open Global Hotkey Window

### ISSUE THREAD

**Issue by Ssiswent** [warpdotdev/Warp#2795]
### Describe the bug

I have checked `Accessibility` and tick the checkbox to grant Warp access.

And I have tried many keyboard shortcuts, they all can't work.

I've switched to English Input, still can't work.

### To Reproduce

1. Open Warp
2. `Settings > Features`
3. Set shortcuts

![Uploading image.png…]()

### Operating System

MacOS

### OS Version

13.0

### Warp Version

v0.2023.03.07.08.02.stable_03

### Does this block you from using Warp daily?

No

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by OoguroxRyuuya** [comment]
感觉 Warp 近期不会解决这个问题，应该是输入法不是 ABC 导致的，你可以试试只用 ABC，应该就生效了

**Comment by Ssiswent** [comment]
> 感觉 Warp 近期不会解决这个问题，应该是输入法不是 ABC 导致的，你可以试试只用 ABC，应该就生效了

我试过了改成ABC，但是还是不行，其他快捷键都是生效的，除了这个。。

**Comment by dannyneira** [comment]
Hi @Ssiswent Sorry to hear about this, I suspect its due to a lack of International input support #341

I can bring this up with the team and we'll post any updates here on this thread.

**Comment by Ssiswent** [comment]
> Hi @Ssiswent Sorry to hear about this, I suspect its due to a lack of International input support #341
> 
> I can bring this up with the team and we'll post any updates here on this thread.

Thanks for your reply, I tried switch to `ABC` input, but it still can't work, so I guess that it's not the main problem.
btw, even if I am using Chinese input, the `cmd + p` shortcut can also work.

**Comment by gaowhen** [comment]
> 感觉 Warp 近期不会解决这个问题，应该是输入法不是 ABC 导致的，你可以试试只用 ABC，应该就生效了

same issue, but switching to `ABC` input resolves it for now.

And it seems like there are some limitations for the key, I tried to set it to `CTRL + ESC`, but failed.

**Comment by Ssiswent** [comment]
> > > 感觉 Warp 近期不会解决这个问题，应该是输入法不是 ABC 导致的，你可以试试只用 ABC，应该就生效了
> > 
> > 
> > 我试过了改成 ABC，但是还是不行，其他快捷键都是生效的，除了这个。。
> 
> 试试先把其他输入法移除，再设置快捷键，应该是有效果的，我试过

![image](https://user-images.githubusercontent.com/37449310/229017152-baf6f853-c80c-45ae-ba80-d57a4c6989d9.png)
![image](https://user-images.githubusercontent.com/37449310/229017325-5f4b1570-ca36-4d2c-953d-ea63bab03b98.png)
还是不行。。。

**Comment by DIYgod** [comment]
Set the input sources to only ABC, then set the hotkey again, then restart Wrap, then it works for me.

**Comment by liuxingyu521** [comment]
> Set the input sources to only ABC, then set the hotkey again, then restart Wrap, then it works for me.

yeah! This also works well for me now. Then I tried to toggle input source to Chinese input，it doesn't work again. It seems only ABC input can in effect.

**Comment by Hoofoo-WHU** [comment]
> 

it's resolved my issue

**Comment by Hoofoo-WHU** [comment]
> 感觉 Warp 近期不会解决这个问题，应该是输入法不是 ABC 导致的，你可以试试只用 ABC，应该就生效了
That's resolved my issue! 👍🏻

**Comment by lotusk** [comment]
我只要切回中文输入法, 就完了. 换回abc 又能弄出来了 疯了.

**Comment by PeterChen1997** [comment]
same problem... That's not cool

**Comment by KaitoMuraoka** [comment]
same problem.
This issue still seems to be unresolved. 😕


It seems possible to forcibly start it by setting a Hotkey on the launcher app called [Raycast](https://www.raycast.com/).

However, I want to start Wrap by itself.

**Comment by silent-tan** [comment]
Why is it that item2 works properly while wrap does not suffice?

**Comment by sonald** [comment]
+1 
this problem is really annoying, this is a killer problem

**Comment by sonald** [comment]
so, no one cares about this problem?

**Comment by xinx1n** [comment]
> Set the input sources to only ABC, then set the hotkey again, then restart Wrap, then it works for me.

But when you switch to another input method, it doesn't work.

**Comment by jiangyuan** [comment]
same problem...

**Comment by peanutgao** [comment]
same problem...

**Comment by linkscope** [comment]
This is a serious problem, and if you can only use it with ABC input, it means that all non-native English developers will have this problem. It's also hard for people who are used to using shortcuts to call the terminal, and I don't think many people would use `commond + Tab` to switch between terminals.

**Comment by XTeam-Wing** [comment]
It has been 2 years, and this problem has not been solved. Uninstall it immediately.

**Comment by xdlrt** [comment]
Unfortunately, this problem is extremely critical for me.

**Comment by cswangs** [comment]
Unfortunately, this problem is extremely critical for me too.

**Comment by liaohaoxiang** [comment]
Unfortunately, this problem is extremely critical for me too.

**Comment by cswangs** [comment]
Great! The issue is fixed, but the hotkey window still shows on a fixed screen instead of the cursor's location, even with the 'active screen' option enabled. It would be great if this could work like iTerm2's hotkey feature. Thanks!

![Image](https://github.com/user-attachments/assets/301838e1-aad9-4580-8965-3d79dc609c82)
